docs/concepts/manifest-builds.md

```diff
@@ -134,6 +137,32 @@ What this means in practice is:
 - Man pages should be placed in `$out/share/man`.
 - Libraries should be placed in `$out/lib`
 
+### Sandboxed builds
+
+The script that you specify in `command` is run on your host machine by default.
+This means that it may build against dependencies that are outside of your
+environment and the artifact may not function correctly when executed or rebuilt
+from another machine.
+
+In order to improve the reproducibility of your build you can specify that it is
+run within a sandbox that doesn't have access to the host machine:
+
+```toml
+[build.myproject]
+sandbox = "pure"
```

Comment:
suggestion: I'd like to draw more attention to this line.

```suggestion
sandbox = "pure" # Enforces the build to only use packages from the environment
```

Reply:
"packages" might be a too narrow distinction but I'll add some variation of yours and Zach's suggestion 👍🏻